src/blacklist.txt

```diff
@@ -4,11 +4,15 @@
 *://coin-hive.com/captcha*
 wss://*.coinhive.com/proxy*
 wss://*.coin-hive.com/proxy*
+
 *://jsecoin.com/server*
 *://*.jsecoin.com/server*
+*://server.jsecoin.com/save/
+*://load.jsecoin.com/request/
```

Comment:
I would rather blacklist the entire sub domain in this case, like `*://server.jsecoin.com/*` and `*://load.jsecoin.com/*`.